mygallery.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>My Gallery</title>
+    <style>
+      body {
+        font-family: Arial, sans-serif;
+      }
+      h1 {
+        text-align: center;
+        color: blue;
+      }
+      .gallery-container {
+        display: flex;
+        flex-wrap: wrap;
+        justify-content: center;
+        margin-top: 20px;
+      }
+      .photo-thumbnail {
+        width: 300px;
+        height: 300px;
+        margin: 20px;
+      }
+      .photo-thumbnail img {
+        width: 100%;
+        height: 100%;
+        object-fit: cover;
+        border-radius: 5px;
+      }
+      header {
+  background-color: #333;
+}
+
+header nav ul {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  display: flex;
+  justify-content: space-between;
+}
+
+header nav li {
+  display: inline-block;
+  background-color: #333;
+  color: #fff;
+  padding: 10px 20px;
+  text-align: center;
+}
+
+header nav a {
+  color: #fff;
+  text-decoration: none;
+}
+
+    </style>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <ul>
+          <li><a href="index.html">Home</a></li>
+          <li><a href="about.html">About</a></li>
+          <li><a href="contact.html">Contact</a></li>
+          <li><a href="mygallery.html">My Gallery</a></li>
+        </ul>
+      </nav>
+    </header>
+    
+    <h1>My Gallery</h1>
+    <div class="gallery-container">
+      <div class="photo-thumbnail">
+        <img src="298633121_170633715530467_5899080429385164709_n.jpg" alt="Photo 1">
```

Comment:
maintain a proper naming for images, and have a proper folder structure like images, all the imahes should be in images folder and same for the css too

Reply:
Done